Source/WebCore/dom/SlotAssignment.cpp

```diff
@@ -308,6 +308,12 @@ void NamedSlotAssignment::didChangeSlot(const AtomString& slotAttrValue, ShadowR
     slot->assignedNodes.clear();
     m_slotAssignmentsIsValid = false;
 
+    for (auto& slot : m_slots.values()) {
```

Comment:
Why are we invalidating all slots? Surely we just need invalidate `slot` here?

Reply:
Wouldn't you need to invalidate the old slot as well? I wasn't sure how to get that one, so I just looped through all of them

Reply:
In that case, we'd also call `didChangeSlot` on the old slot. See `hostChildElementDidChangeSlotAttribute`.